### DISCUSSION TITLE
How to check costs per subdomain

### DISCUSSION THREAD

**Discussion by yukipioa** [mozilla/hubs#5302] [Hubs Cloud Questions] [answered]
Hello.
I have a question.

I would like to check which subdomain is costing how much. Where and how could I check?
For example, the subdomain "https://aaaa/bbbb" costs low and the subdomain "https://aaaa/cccc" costs high.

Many Thanks!

**Comment by pattersonbl2** [comment, marked as answer]
Are you talking about AWS cost or renewing your domain cost?  These subdomains wouldn't have a divided cost.

**Reply by yukipioa** [reply]
@Pattersonbl2 
Thank you for replying!
I mean AWS costs such as data traffic costs.
You are saying that the costs are not separated per subdomain, so we cannot check per subdomain, is that correct?

**Reply by pattersonbl2** [reply]
that is correct.  This article can help understand data transfer costs. 
 https://aws.amazon.com/blogs/architecture/overview-of-data-transfer-costs-for-common-architectures/
